scripts/measurements/tools_run.sh

```diff
@@ -8,37 +8,34 @@
 PATH=$PATH:/opt/bro/bin
 export PATH
 
+truncate -s 0 performance.log
 echo -e "bro performance\n" >> performance.log
-for pcapFile in `ls Test_pcap_files/sample_capture_test*.pcap`
+for pcapFile in `sudo ls data/packet/*.pcap`
 do
 
     #perform a trial run to get all the data into RAM
-    /usr/bin/time bro -r $pcapFile > /dev/null 2>&1
+    sudo /usr/bin/time bro -r $pcapFile > /dev/null 2>&1
```

Comment:
Running bro without sudo gives an error.

Reply:
Also, sir, this script must be run as a non-root user because of Wireshark. Due to running it as a non-root user, there is a permission denied error when trying to write into the performance.log file. This can be removed using chmod command. So, can I include this command in this script?

Reply:
bro should run without sudo. Can you please check it again to see if the file permissions is the issue here?